src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -651,25 +456,8 @@ void InitSurfaceHeatBalance(EnergyPlusData &state)
     if (state.dataHeatBalSurfMgr->InitSurfaceHeatBalancefirstTime) DisplayString(state, "Initializing Solar Heat Gains");
 
     InitSolarHeatGains(state);
-    if (state.dataEnvrn->SunIsUp && (state.dataEnvrn->BeamSolarRad + state.dataEnvrn->GndSolarRad + state.dataEnvrn->DifSolarRad > 0.0)) {
```

Comment:
Moved to `DaylightingManager::manageDaylighting`.

Reply:
This is a really nice cleanup in the HBSM.